apps/themebuilder/app/_components/appearance-toggle/appearance-toggle.tsx

```diff
@@ -48,8 +48,6 @@ export const AppearanceToggle = ({
           variant={scheme.value === colorScheme ? 'primary' : 'secondary'}
           data-color='neutral'
           aria-label={`${t('appearanceToggle.set-to')} ${scheme.name} ${t('appearanceToggle.view')}`}
-          // biome-ignore lint/a11y/useSemanticElements: explanation needed
-          role='radio'
```

Comment:
Don't we want this here?

Reply:
Biome did not get triggered in `ToggleGroup` (because the role is set progtrammatically) but if i add role="radio" there manually it _still_ is not triggered weirdly. 
But more importantly I just realized i have been looking at the wrong theme button, this is of course the inner toggle and not the global one lol. So radio does make sense after all. I'll put it back with the new biome-ignore syntax 👍